src/components/Photo/DocumentLiveCapture.tsx

```diff
@@ -11,48 +12,52 @@ import Camera from '../Camera'
 import CameraError from '../CameraError'
 import style from './style.scss'
 
-/* type State = {
-  hasBecomeInactive: boolean,
-  hasCameraError: boolean,
-  isCapturing: boolean,
-}
+import type { SdkMetadata } from '~types/commons'
+import type { WithLocalisedProps, WithTrackingProps } from '~types/hocs'
+import type { CapturePayload } from '~types/redux'
+import type { HandleCaptureProp, RenderFallbackProp } from '~types/routers'
+import type { DocumentTypes } from '~types/steps'
 
 type Props = {
-  translate: (string, ?{}) => string,
-  onCapture: Function,
-  renderFallback: Function,
-  isUploadFallbackDisabled: boolean,
-  trackScreen: Function,
-  documentType: string,
-  className: string,
-  containerClassName: string,
-  renderTitle: Function,
-  renderError: Function,
-} */
-
-const IDEAL_CAMERA_HEIGHT_IN_PX = 1080
-const FALLBACK_HEIGHT_IN_PX = 720
-
-export default class DocumentLiveCapture extends Component {
-  webcam = null
+  className?: string
+  containerClassName: string
+  documentType?: DocumentTypes
+  isUploadFallbackDisabled: boolean
+  onCapture: HandleCaptureProp
```

Comment:
Minor: Why not give the same name for the type & function for consistency?

Reply:
Sorry what did you mean? Is that `onCapture: HandleCaptureProp` with `SomeCallbackProp` naming issue?

Reply:
Not really. Just figured it would be nicer if it's `handleCapture:  HandleCaptureProp`. A minor detail TBH.